src/packages/core/property/conditions/manifests.ts

```diff
@@ -0,0 +1,10 @@
+import type { ManifestTypes } from '@umbraco-cms/backoffice/extension-registry';
+
+export const manifests: Array<ManifestTypes> = [
+	{
+		type: 'condition',
+		name: 'Writable Property Condition',
+		alias: 'Umb.Condition.Property.Writable',
```

Comment:
It can use `UMB_WRITABLE_PROPERTY_CONDITION_ALIAS` constant here.

Reply:
Thanks 💯  In have updated the PR.